openslides/motions/views.py

```diff
@@ -178,7 +179,8 @@ def update(self, request, *args, **kwargs):
         # Check permissions.
         if (not has_perm(request.user, 'motions.can_manage') and
                 not (motion.is_submitter(request.user) and motion.state.allow_submitter_edit) and
-                not has_perm(request.user, 'motions.can_see_and_manage_comments')):
+                not has_perm(request.user, 'motions.can_see_comments') and
```

Comment:
Bool combination is wrong, should be (not (has_perm x and has_perm y)).

Reply:
Done.